### PR TITLE
increase minStake

### PR DESCRIPTION
**Pull request by Vitomir2**
# Description

- initialize the constants as variables and set minStake to big int of 15k Hydra;

# Changes include

- [x] New feature (non-breaking change that adds functionality)

# Checklist

- [x] I have added at least 1 reviewer
- [x] I have added the relevant labels
- [x] I have updated the official documentation

## Testing

- [x] I have tested this code manually

### Manual tests

I have executed the `register-validator` command which successfully registered the validator but was not able to stake when the amount was less than the minimum stake. It reverted with the corresponding custom error. Then, I was successfully able to stake with more than the minimum. Additionally, tested the happy case and it worked like a charm.

### CODE REVIEW

File: consensus/polybft/contracts_initializer.go [hunk 1]
```diff
@@ -12,12 +12,12 @@ import (
 	"github.com/umbracle/ethgo/abi"
 )
 
-const (
+var (
 	// Hydra modification: These are the minimal values that the ValidatorSet contract accept
```
Comment (R-Santev) [inline consensus/polybft/contracts_initializer.go:16]: These are not the minimal values accepted anymore.
To make it clear, let's make the variable `initialMinStake` and let's add a constant minStakeLimit to be used everywhere in the tests, so we can know all operations work properly with the minimum min stake possible.

**Review by R-Santev — CHANGES_REQUESTED** [review]

File: h_docs/polybft_setup.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ I am describing our custom process, because it is different.
 We need to set native token to be mintable, so we can premine balances to different addresses
 
 ```
-./hydra genesis --block-gas-limit 10000000 --epoch-size 10 --validators-path ./ --validators-prefix test-chain- --consensus polybft --native-token-config Hydra:HDR:18:true:0x211881Bb4893dd733825A2D97e48bFc38cc70a0c --reward-wallet 0x61324166B0202DB1E7502924326262274Fa4358F:1000000 --premine 0x211881Bb4893dd733825A2D97e48bFc38cc70a0c:70000000000000000000000 --premine 0xdC3312E368A178e24850C6dAC169646c5fD14b93:700000000000000000000 --proxy-contracts-admin 0x211881Bb4893dd733825A2D97e48bFc38cc70a0c --chain-id 8844
+./hydra genesis --block-gas-limit 10000000 --epoch-size 10 \ --validators-path ./ --validators-prefix test-chain- --consensus polybft --native-token-config Hydra:HDR:18:true:0x211881Bb4893dd733825A2D97e48bFc38cc70a0c   --reward-wallet 0x61324166B0202DB1E7502924326262274Fa4358F:1000000 --premine 0x211881Bb4893dd733825A2D97e48bFc38cc70a0c:70000000000000000000000 --premine 0xdC3312E368A178e24850C6dAC169646c5fD14b93:30000000000000000000000 --proxy-contracts-admin 0x211881Bb4893dd733825A2D97e48bFc38cc70a0c --chain-id 8844
```
Comment (R-Santev) [inline h_docs/polybft_setup.md:33]: Why are `\` and an empty space on the next line added?

**Review by R-Santev — CHANGES_REQUESTED** [review]
Just one small note

File: h_docs/polybft_setup.md (hunk 2 above)
Comment (Vitomir2) [reply h_docs/polybft_setup.md:33]: Well, it is for sure by mistake. Will delete it

**Review by R-Santev — APPROVED** [review]
